src/ansible_navigator/configuration_subsystem/transform.py

```diff
@@ -7,13 +7,67 @@
 from typing import Dict
 from typing import List
 from typing import Tuple
+from typing import Union
 
 from ..utils.compatibility import importlib_resources
+from ..utils.functions import dict_merge_in_place_list_replace as dict_merge
```

Comment:
Rather than giving it a crazy name and aliasing, I'd just rename the function.

Reply:
There are so many ways to merge dicts, net-new, append lists, this is why the name is what it is.....

Let me think...........

Reply:
Dedicated files! much cleaner